krs/groups.py

```diff
@@ -321,12 +322,19 @@ def main():
     parser_remove_user_group.add_argument('username', help='username of user')
     parser_remove_user_group.add_argument('group_path', help='group path (/parentA/parentB/name)')
     parser_remove_user_group.set_defaults(func=remove_user_group)
+    parser_modify = subparsers.add_parser('modify', help='modify an existing group')
+    parser_modify.add_argument('group_path', help='group path (/parentA/parentB/name)')
+    parser_modify.add_argument('--new-group-path', metavar='PATH', help='change group path (/parentA/parentB/name)')
+    parser_modify.add_argument('attrs', nargs='?', help='set attributes from JSON dictionary, e.g. \'{"foo"=[1,2,3], "bar"="baz"}\'')
```

Comment:
For the users, I used
```python
parser_modify.add_argument('attribs', nargs=argparse.REMAINDER)
```

This allows you to do direct key=value on the command line, but may not be very good with multiple values
```python -m krs.groups modify /my/path attr1=1 attr1=2```
would produce `["attr1=1", "attr1=2"]` which would have to be parsed.

The json method might be better.  Just wanted to raise this to think about.